lib/visualize_packwerk/package_relationships.rb

```diff
@@ -69,21 +50,18 @@ def draw_graph!(graph, highlighted_node_names, show_all_nodes: false)
       # Create graph nodes
       graphviz_nodes = T.let({}, T::Hash[String, GraphViz::Node])
 
-      nodes_to_draw = graph.nodes
+      nodes_to_draw = graph.nodes.select{|n| node_names.include?(n.name) }
 
       nodes_to_draw.each do |node|
-        next unless highlighted_node_names.any? { |highlighted_node_name| node.depends_on?(highlighted_node_name) } || highlighted_node_names.include?(node.name)
```

Comment:
Before, we showed one level of ancestors and highlighted the given packs.

Instead, we *only* want to render exactly what the user is passing in. Later, we can think of a way to specify how many levels of ancestors/descendants to render.